packages/beacon-node/test/e2e/doppelganger/doppelganger.test.ts

```diff
@@ -88,7 +89,7 @@ describe.skip("doppelganger / doppelganger test", function () {
 
     const validatorUnderTest = validatorsWithDoppelganger[0];
     const pubKey = validatorUnderTest.validatorStore.votingPubkeys()[0];
-    const beaconBlock = ssz.allForks.phase0.BeaconBlock.defaultValue();
+    const beaconBlock = phase0SSZ.BeaconBlock.defaultValue();
```

Comment:
can't we do something like `ssz.phase0.BeaconBlock`? This looks strange and is semantically completely different to what we are used to

Reply:
Yes we can do that. The upcoming PRs will refactor the ssz types and will be part of that scope.

Reply:
we can just do the following here already

```suggestion
    const beaconBlock = ssz.phase0.BeaconBlock.defaultValue();
```

in the previous code (on unstable), to use `allForks.phase0` here was incorrect